ads/google/a4a/utils.js

```diff
@@ -110,7 +110,8 @@ export function googleAdUrl(
   const referrerPromise = viewerForDoc(a4a.getAmpDoc()).getReferrerUrl();
   return getAdCid(a4a).then(clientId => referrerPromise.then(referrer => {
     const adElement = a4a.element;
-    const slotNumber = adElement.getAttribute('data-amp-slot-index');
+    window.ampAdGoogleIfiCounter = window.ampAdGoogleIfiCounter || 1;
```

Comment:
I assume we do not need to quote ampAdGoogleIfiCounter meaning it would not be renamed or if renamed, it would be equivalent across extension binaries?